src/MoleculeToFragments.cc

```diff
@@ -72,21 +79,29 @@ namespace {
     return temp_mol;
   }
 
-  Molecule bond_rotate(const Molecule& mol, int bond_id, fltype th) {
+  /**
+   * @brief Rotate the bond that connects the two fragments.
+   * 
+   * @param mol Molecule to be rotated
+   * @param bond_id Bond ID that serves as the axis of rotation
+   * @param th Rotation angle in radian
+   * @param rotate_is_id1 Whether fragment is rotated around atom_id1 or atom_id2 (default: true)
```

Comment:
rotateは動詞なので、is はおかしいと思います ＆ rotateするのはatom id1 が属するフラグメントであって原子 atom_id1 自体ではない（こいつは回転しても座標は変わらない）と思います。

> @param rotate_atom1_side
> Specifies which substructure to rotate around the bond. If set to true, the substructure containing `atom_id1` is rotated; otherwise, the substructure containing `atom_id2` is rotated.

…chatGPT先生にお願いしたら結構長くなっちゃいました。